data/mods/CRT_EXPANSION/items/crt_toolarmor.json

```diff
@@ -63,7 +63,14 @@
     "coverage": 100,
     "encumbrance": 5,
     "warmth": 75,
-    "artifact_data": { "charge_type": "ARTC_SOLAR", "effects_worn": [ "AEP_ALL_UP", "AEP_INT_DOWN" ] },
+    "//": "Legacy artifact data is set for all stats +2 except INT charge_type solar",
+    "relic_data": {
```

Comment:
![image](https://user-images.githubusercontent.com/30208401/89468668-f9b00880-d745-11ea-9772-5ac85adcc1cb.png)
doesn't seem to work.
![image](https://user-images.githubusercontent.com/30208401/89468678-016fad00-d746-11ea-9b65-8f4d543cd819.png)

Reply:
I'm not sure what change exactly fixed it but I spawned one in and equipped it and it gave the stat increases.